jekyll/features.html

```diff
@@ -9,19 +9,32 @@ <h1 class="text-centered page-title main-heading">Features</h1>
   <section class="content">
     <div class="pure-g">
       <div class="pure-u-1">
-        <h1>Intuitive and clean syntax</h1>
+        <h1>One language to rule them all</h1>
       </div>
       <div class="pure-u-1 pure-u-md-1-2 code-align">
-        <h2>
-          Taking inspiration from multiple languages, the Nim syntax is
-          easily understood and results in code that can be modified very
-          effectively.
-        </h2>
+        <h2>Good for everything, from shell scripting to web front and backend,
+          to machine learning, high-performance computing (HPC), and embedded.</h2>
```

Comment:
```suggestion
        <h2>Multipurpose, web backend and frontend, machine learning,        
        high-performance computing, embedded, IoT, WebAssembly, scripting.</h2>
```


"Good for everything" sounds like "jack of all trades master of none"

Reply:
"general purpose" ?

Reply:
I think the slogan is just like "life is short, use python". It is easy to be remembered and very expressive.

Reply:
So if you want a long life you should NOT use Python, interestin' 🤔

Reply:
lol, the translated version("life is short, use python") is famous in my country.

https://www.programmersought.com/article/1514161951